### ISSUE TITLE
In progress shows default to 5 min mark

### ISSUE THREAD

**Issue by grower-rhyme** [jellyfin-archive/jellyfin-roku-legacy#717]
**Describe the bug**
When I begin a new episode of a show and exit without completing the episode, the progress marker seems to default to 5 minutes in regardless of how early or far in I am.

**To Reproduce**

1. Start a TV show
2. Watch any length of time
3. Close Jellyfin
4. Go back to Jellyfin to resume the show
5. The progress bar will say you're at 5 mins into the episode.

**Expected behavior**

The show should resume close to where you left off, give or take a min or so.

**Logs**

I'm not sure what logs you would need, but I'm more than happy to share any I have if needed. I'm not the most tech savvy in the world so you may have to let me know where to find them (I'm hosting on Debian, if that helps).

**Comment by Zerva5** [comment]
I have experienced this as well on the stable release available on the Roku channel store (whatever it's called). 

I found it only happened when exiting directly to the home page by pressing the home button while playing a show, if you press the back button and go to the Jellyfin home page first and then exit it saves the correct time. 

I've been playing around with the dev branch a bit and haven't been able to reproduce it. I assume you're experiencing this bug on the version from the Roku channel store and not the development version @grower-rhyme?

This may have already been fixed and just hasn't made its way up stream yet. If I run into it on the dev branch I'll try to fix it and submit a pull request.
